src/store/utils.ts

```diff
@@ -48,28 +48,24 @@ export function calcFolderAbsParentLevelAndPath(
 }
 
```

Comment:
Can you add a short comment in the code to explain why you decided to use the vanilla reduce? (so for future instance we know why is there)

Reply:
done